sirepo/nersc.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""Run Python processes in background
+
+:copyright: Copyright (c) 2016 RadiaSoft LLC.  All Rights Reserved.
+:license: http://www.apache.org/licenses/LICENSE-2.0.html
+"""
+import re
+import subprocess
+from pykern import pkconfig
+from pykern.pkcollections import PKDict
+
+
+_ACCOUNT_NOT_FOUND = "no such fileset"
+
+VALID_TEST_ACCOUNT = "VALID_TEST_ACCOUNT"
+
+
+def assert_project(project):
+    if not project:
+        return ""
+    res = (
+        _test_res(project)
+        if pkconfig.channel_in_internal_test()
+        else subprocess.run(
+            ("hpssquota", project),
+            stdout=subprocess.PIPE,
+            stderr=subprocess.STDOUT,
+            text=True,
+        )
+    )
+    assert not re.search(_ACCOUNT_NOT_FOUND, res.stdout), invalid_project_msg(project)
+    return sbatch_account(project)
+
+
+def sbatch_account(project):
+    return f"#SBATCH --account={project}"
+
+
+def invalid_project_msg(project):
```

Comment:
doesn't need to be exported and would be just fine inline.

Reply:
[f5c7773](https://github.com/radiasoft/sirepo/pull/5493/commits/f5c7773f6e8928ded548737d1aacb5fb99dd8471)